client/my-sites/checklist/checklist-show/index.jsx

```diff
@@ -136,12 +137,15 @@ class ChecklistShow extends PureComponent {
 		const completed = tasks && ! find( tasks, { completed: false } );
 
 		let title = 'Site Checklist';
+		let path = '/checklist/:site';
```

Comment:
You can retrieve the current path by calling [`getRouteHistory` selector](https://github.com/Automattic/wp-calypso/blob/master/client/state/ui/action-log/selectors.js#L38-L40). `ui.actionLog` state contains the history of `ROUTE_SET` actions which looks like:

```
actionLog: [
      {
        type: 'ROUTE_SET',
        path: '/',
        query: {},
        timestamp: 1520316229544
      },
      {
        type: 'ROUTE_SET',
        path: '/stats/day/taggon.wordpress.com',
        query: {},
        timestamp: 1520316237110
      },
      {
        type: 'ROUTE_SET',
        path: '/stats/day/taggontest419273.wordpress.com',
        query: {},
        timestamp: 1520316249063
      },
      {
        type: 'ROUTE_SET',
        path: '/checklist/taggontest419273.wordpress.com',
        query: {},
        timestamp: 1520316252014
      }
```

You will see the last `ROUTE_SET` log points the current path.

Reply:
Oh, I was confused. Never mind and ship it.